pct.sh

```diff
@@ -15,6 +15,7 @@ fi
 
 exec java \
 	-jar target/pct.jar \
+	test-plugins \
```

Comment:
The upstream PCT JAR now has two modes: `list-plugins` (to divide the list by repository) and `test-plugins` (the old mode).